apps/meteor/app/e2e/server/methods/updateGroupKey.ts

```diff
@@ -34,8 +34,8 @@ Meteor.methods<ServerMethods>({
 			}
 
 			// uid also has subscription to this room
-			const { modifiedCount } = await Subscriptions.setGroupE2ESuggestedKey(uid, rid, key);
-			if (modifiedCount) {
+			const { value } = await Subscriptions.setGroupE2ESuggestedKey(uid, rid, key);
+			if (value) {
 				void notifyOnSubscriptionChangedByRoomIdAndUserId(rid, uid);
```

Comment:
looks like we could use `notifyOnSubscriptionChanged(value)` here as well

Reply:
there u go